bin/node/node/src/service.rs

```diff
@@ -16,14 +16,24 @@
 
 //! Service and ServiceFactory implementation. Specialized wrapper over substrate service.
 
+// =====================================================================================
+// =====================================================================================
+// =====================================================================================
```

Comment:
I think the update guide section is useful, but tbh I'm not a fan of these. They add a lot of visual noise when CAPS do the trick here

Reply:
The idea is to actually make a visual noice :)))

Reply:
I think sections like this should only be used in places where a critical amount of attention is needed. Here there's not much else happening, so CAPS suffice to draw attention. Worst case we merge to master, and quickly realize our syncing doesn't work because we can't connect to the RPCs